pyproject.toml

```
@@ -29,7 +29,7 @@ dependencies = [
     "uvicorn",
     "click",
     "boto3",
-    "ga4gh.vrs==2.0.0a14",
+    "ga4gh.vrs~=2.0.0",
```

Or could do `"ga4gh.vrs~=2.0",`

```suggestion
    "ga4gh.vrs==2.*",
```